src/modules/fstab/main.py

```diff
@@ -158,11 +166,19 @@ def generate_crypttab_line_info(self, partition):
         if not mapper_name or not luks_uuid:
             return None
 
+        # Set crypttab password for partition to none and remove crypttab options
+        # on root partition when /boot is unencrypted
+        password = "/crypto_keyfile.bin"
+        crypttab_options = self.crypttab_options
+        if partition["mountPoint"] == "/" and unencrypted_separate_boot:
+            password = 'none'
+            crypttab_options = ''
```

Comment:
All the other strings used in Calamares Python modules (rare exceptions where they must contain a `"`) are `"` delimited. For consistency, please change that (but this isn't a blocker).